.github/workflows/terraform_checks.yml

```diff
@@ -26,27 +26,27 @@ jobs:
 
   check-terraform-validity:
     runs-on: ubuntu-latest
+    env:
+      TERRAFORM_DIRS: |
+          dev dev/persistent test test/persistent demo demo/persistent training training/persistent
+          stg stg/persistent pentest pentest/persistent prod prod/persistent
+          global
     steps:
       - uses: actions/checkout@v2
       - uses: hashicorp/setup-terraform@v1
         with:
           terraform_version: 0.15.1
       - name: Terraform Init
         run: |
-          pushd dev; pushd persistent; terraform init -backend=false; popd; terraform init -backend=false; popd
-          pushd test; pushd persistent; terraform init -backend=false; popd; terraform init -backend=false; popd
-          pushd demo; pushd persistent; terraform init -backend=false; popd; terraform init -backend=false; popd
-          pushd stg; pushd persistent; terraform init -backend=false; popd; terraform init -backend=false; popd
-          pushd prod; pushd persistent; terraform init -backend=false; popd; terraform init -backend=false; popd
-          pushd pentest; pushd persistent; terraform init -backend=false; popd; terraform init -backend=false; popd
-          pushd global; terraform init -backend=false;
-
+          for d in $TERRAFORM_DIRS
+          do
+            echo "Initializing $d";
+            (cd $d && terraform init -backend=false)
```

Comment:
```suggestion
            terraform -chdir=$d init -backend=false
```

Reply:
My slight preference is to not use `chdir` and keep the `cd &&` syntax - I actually find it a little easier to read the subsequent `terraform` command since it has less flags in it.

Reply:
Fine by me!